### PR TITLE
Fix: overflow issues on leveldb file picker

### PR DESCRIPTION
**Pull request by jjlumagbas**
Solved the issue by abandoning the native input file picker (since we're not using it anyway, in favor of Electron's dialog) and just styling a grouped input using a div.

### CODE REVIEW

File: src/pages/Home.vue [hunk 1]
```diff
@@ -78,72 +78,52 @@ form {
   flex-direction: column;
 }
 
-.file {
-  position: relative;
-  display: inline-block;
+input,
+button {
+  font-family: 'Fira Mono', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
```
Comment (jjlumagbas) [inline src/pages/Home.vue]: This is a kludge. I can't figure out why the input and buttons don't inherit from body font?

Comment (emanlodovice) [reply src/pages/Home.vue]: any ideas @arnellebalane ?

Comment (arnellebalane) [reply src/pages/Home.vue]: @jjlumagbas @emanlodovice That's default behaviour. In the global styles you can add 

```css
input,
button {
  font: inherit;
  color: inherit;
}
```

Comment (jjlumagbas) [reply src/pages/Home.vue]: Applied this change in 22e179e

**Comment by jjlumagbas** [comment]
I may have introduced merging conflicts: I first rebased from latest master, and then pull-rebased from origin/bugfix/file-picker. Not sure if I merged correctly, please double check?

**Comment by arnellebalane** [comment]
@jjlumagbas I think your rebases didn't work properly. You might have to cherry-pick your commits in this case.

**Comment by jjlumagbas** [comment]
Ok, I think I've resolved conflicts from my end, but not sure if all of changes from @emanlodovice are reflected in the latest commit.

**Comment by emanlodovice** [comment]
@jjlumagbas  I think the commits are messed up. It might be better to scrap this branch and make a new one from scratch.

**Comment by arnellebalane** [comment]
@jjlumagbas @emanlodovice I've cherry-picked the relevant commits from this PR into GH-13. Closing this PR.

**Comment by jjlumagbas** [comment]
Thanks @arnellebalane, sorry guys this project has fallen behind other priorities for now

**Pull request closed without merging**